### PR TITLE
🌐 Add French translation for `docs/fr/docs/contributing.md`

### PR DESCRIPTION
**Pull request by JulianMaurin**
Hi :v:

This is a PR to translate docs/contributing.md to french 🇫🇷

Thanks 🦩

### CODE REVIEW

File: docs/fr/docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,501 @@
...
+!!! tip
+    Vous pouvez <a href="https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request" class="external-link" target="_blank">ajouter des commentaires avec des suggestions de changement</a> aux pull requests existantes.
+
+    Consultez les documents concernant <a href="https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-request-reviews" class="external-link" target="_blank">l'ajout d'un review de pull request</a> pour l'approuver ou demander des modifications.
+
+* Vérifiez dans <a href="https://github.com/tiangolo/fastapi/issues" class="external-link" target="_blank">issues</a> pour voir s'il y a une personne qui coordonne les traductions pour votre langue.
+
+* Ajoutez une seule pull request par page traduite. Il sera ainsi beaucoup plus facile pour les autres de l'examiner.
+
+Pour les langues que je ne parle pas, je vais attendre que plusieurs autres reviews de la traduction avant de merge.
```
Comment (NastasiaSaby) [inline docs/fr/docs/contributing.md:283]: I think it should be "je vais attendre plusieurs autres reviews de la traduction"

Comment (JulianMaurin) [reply docs/fr/docs/contributing.md:283]:
```suggestion
Pour les langues que je ne parle pas, je vais attendre plusieurs autres reviews de la traduction avant de merge.
```

File: docs/fr/docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,501 @@
...
+
+### Documentation pour les tests
+
+La plupart des tests sont en fait effectués par rapport aux exemples de fichiers sources dans la documentation.
+
+Cela permet de s'en assurer que:
+
+* La documentation est à jour.
+* Les exemples de documentation peuvent être exécutés tels quels.
+* La plupart des fonctionnalités sont couvertes par la documentation, assurée par la couverture des tests.
```
[the remark this answers is not in the document]
Comment (JulianMaurin) [reply docs/fr/docs/contributing.md:209]:
```suggestion
* La plupart des fonctionnalités sont couvertes par la documentation, assurées par la couverture des tests.
```

File: docs/fr/docs/contributing.md [hunk 3]
```diff
@@ -0,0 +1,501 @@
...
+
+En fait, ces blocs de code ne sont pas écrits à l'intérieur du Markdown, ce sont des fichiers Python dans le répertoire `./docs_src/`.
+
+Et ces fichiers Python sont inclus/injectés dans la documentation lors de la génération du site.
+
+### Documentation pour les tests
+
+La plupart des tests sont en fait effectués par rapport aux exemples de fichiers sources dans la documentation.
+
+Cela permet de s'en assurer que :
```
Comment (JulianMaurin) [inline docs/fr/docs/contributing.md:205]:
```suggestion
Cela permet de s'assurer que :
```

**Review by rjNemo — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 9a4e4abc71df4f2326e268ec328ca34a15f9d4c6 at: https://649a0af297baca0a929a9e7b--fastapi.netlify.app

**Review by axel584 — APPROVED** [review]

**Comment by tiangolo** [comment]
Thank you @JulianMaurin! 🚀 

And thanks for the reviews @NastasiaSaby, @axel584, @rjNemo 🙇 🍰

**Pull request merged**